### PR TITLE
Dependencies/213

### LINKED ISSUE #213: Update dependencies after API change in bucketfs-java
Dependency update is required as 
https://github.com/exasol/exasol-test-setup-abstraction-java/ as a direct dependency to bucketfs-java as well as to exasol-testcontainers. So both dependencies must be compatible.

### PR DESCRIPTION
**Pull request by ckunki**
fixes #213

### CODE REVIEW

File: doc/changes/changes_6.4.1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Test Containers for Exasol on Docker 6.4.1, released 2022-??-??
+
+Code name: Dependency Upgrade
```
Comment (redcatbear) [inline doc/changes/changes_6.4.1.md:3]:
```suggestion
Code name: Dependency Upgrade on top of 6.4.0
```
Disambiguation

File: doc/changes/changes_6.4.1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Summary
+
+Updated dependencies in particular to enable use in exasol-test-setup-abstraction-java which dependes on bucketfs-java directly as well as transitively via exasol-testcontainers.
```
Comment (redcatbear) [inline doc/changes/changes_6.4.1.md:7]:
```suggestion
Updated dependencies in particular to enable use in `exasol-test-setup-abstraction-java` which depends on `bucketfs-java` directly as well as transitively via `exasol-testcontainers`.
```

File: pom.xml [hunk 3]
```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>bucketfs-java</artifactId>
-            <version>2.4.1</version>
+            <version>2.6.0</version>
```
Comment (redcatbear) [inline pom.xml:78]: I just realized that you introduced a breaking change in the API (ip --> host). I overlooked that in my review. That is actually a version 3.0.0.

File: src/main/java/com/exasol/bucketfs/testcontainers/TestcontainerBucketFactory.java [hunk 4]
```diff
@@ -54,7 +54,7 @@ public synchronized Bucket getBucket(final String serviceName, final String buck
                     .stateRetriever(monitor.createStateRetriever()) //
                     .serviceName(serviceName) //
                     .name(bucketName) //
-                    .ipAddress(this.host) //
+                    .host(this.host) //
```
Comment (redcatbear) [inline src/main/java/com/exasol/bucketfs/testcontainers/TestcontainerBucketFactory.java:57]: This is a breaking change in the API. I think we need to make a new release of BFSJ with a deprecated method `ip` that delegates to `host`.

**Review by redcatbear — APPROVED** [review]

File: pom.xml (hunk 3 above)
Comment (ckunki) [reply pom.xml:78]: I was aware of the change being a breaking one but rated a single renamed method not being worth to increase the major version. As the release has been done I am not sure if we should plan to change the version.

**Review by redcatbear — APPROVED** [review]

**Pull request merged**